fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerIndexFromIndexTest.java

```diff
@@ -1110,4 +1112,94 @@ public void testIndexFromIndexRebuild() {
             context.commit();
         }
     }
+
+    @Test
+    public void testIndexFromIndexBlock() {
+        // start indexing by Index, verify continuation
+        final FDBStoreTimer timer = new FDBStoreTimer();
+        final int numRecords = 49;
+        final int chunkSize  = 12;
+        final int numChunks  = 1 + (numRecords / chunkSize);
+
+        Index srcIndex = new Index("src_index", field("num_value_2"), EmptyKeyExpression.EMPTY, IndexTypes.VALUE, IndexOptions.UNIQUE_OPTIONS);
+        Index tgtIndex = new Index("tgt_index", field("num_value_3_indexed"), IndexTypes.VALUE);
+        FDBRecordStoreTestBase.RecordMetaDataHook hook = myHook(srcIndex, tgtIndex);
+
+        openSimpleMetaData();
+        populateData(numRecords);
+
+        openSimpleMetaData(hook);
+        buildSrcIndex(srcIndex);
+
+        // Partly build index
+        openSimpleMetaData(hook);
+        buildIndexAndCrashHalfway(tgtIndex, chunkSize, 1, timer,
+                OnlineIndexer.IndexingPolicy.newBuilder()
+                        .setSourceIndex("src_index")
+                        .forbidRecordScan()
+                        .build());
+
+        openSimpleMetaData(hook);
+        String luka = "my name is Luka";
```

Comment:
Probably worth giving this a descriptive name like `blockId`.
It might also be good to make this somewhat demonstrative, and use what would be a good blockId in production. such as `Blocked: #1981` or at least `Blocked by Josef`

Reply:
✅